ui/package.json

```diff
@@ -194,8 +191,7 @@
     "pvutils": "^1.0.17",
     "qunit": "^2.19.4",
     "qunit-dom": "^2.0.0",
-    "sass": "^1.58.3",
-    "sass-svg-uri": "^1.0.0",
```

Comment:
this dependency is no longer necessary, per [this comment](https://github.com/hashicorp/vault/pull/26245/files#r1548744545), so i removed it